solarforecastarbiter/datamodel.py

```diff
@@ -45,8 +45,10 @@ class Site:
         Elevation of the Site in meters above mean sea level, e.g. 1007
     timezone : str
         IANA timezone of the Site, e.g. Etc/GMT+8
-    network : str, optional
-        Measurement network name, e.g. SURFRAD
+    site_id : str, optional
+        UUID of the Site in the API
+    provider : str, optional
+        Provider of the Site information.
```

Comment:
In the API provider is the organization that the site belongs to. I think network is too specific; what about non-reference data from a utility? Owner isn't too far off, at least in the sense that it is the owner of the Site object (not necessarily the physical site). 

So reference data will have some Reference organization as the provider, and the network (like UO, MIDC) will be in extra_parameters

Reply:
Ok. Currently the dashboard has "Network (optional)" as its own field rather than in extra_parameters. I think its ok to dump the field, but probably discuss that elsewhere. cc @lboeman